### PR TITLE
Fix cache service coverage for Minion tasks

### PR DESCRIPTION
**Pull request by kraih**
#4526 showed that coverage is not collected correctly for Minion tasks.

### CODE REVIEW

File: t/lib/OpenQA/Test/Utils.pm [hunk 1]
```diff
@@ -93,7 +93,23 @@ sub cache_minion_worker {
...
+                            $worker->on(
+                                dequeue => sub {
+                                    my ($worker, $job) = @_;
+                                    $job->on(
+                                        cleanup => sub {
+                                            Devel::Cover::report() if Devel::Cover->can('report');
+                                        });
+                                });
+                        });
+                });
```
Comment (okurz) [inline t/lib/OpenQA/Test/Utils.pm:112]: one has to admire such prettyness :)

Would this work though

```suggestion
            OpenQA::CacheService::run(
                ['run', '--dequeue-timeout', '1'],
                sub ($app) {
                    $app->minion->on(worker => sub ($minion, $worker) {
                            $worker->on(dequeue => sub ($worker, $job) {
                                    $job->on(cleanup => sub { Devel::Cover::report() if Devel::Cover->can('report') });
                                });
                        });
                });
```

Comment (kraih) [reply t/lib/OpenQA/Test/Utils.pm:112]: Sure, i don't want to covert the whole file to signatures though. 😉

Comment (kraih) [reply t/lib/OpenQA/Test/Utils.pm:112]: Oh, the file is already partially converted.

**Comment by kraih** [comment]
Even after a few iterations, it does not look like the change had the effect i was hoping for. Back to the drawing board i guess.

**Comment by kraih** [comment]
I know the current code is wrong, but i can't prove it with tests yet.

**Pull request closed without merging**